scripts/src/models/translations.ts

```diff
@@ -11,6 +16,30 @@ export class OfferTranslation extends BaseEntity {
 		return (this as typeof BaseEntity).create(data!) as OfferTranslation;
 	}
 
+	public static async getTranslations(criteria: GetTranslationsCriteria = {}): Promise<OfferTranslation[]> {
+		//  todo add cache
+		const languages = criteria.languages;
+		const offerId = criteria.offerId;
+		const paths = criteria.paths;
+		const query = OfferTranslation.createQueryBuilder("translations");
+		if (languages) {
+			query.where("translations.language IN (:languages)", { languages });
+		}
+		if (offerId) {
+			query.andWhere("translations.offer_id = :offerId", { offerId });
```

Comment:
i'm not sure how `andWhere` works, but if the previous predicate isn't true, then the first `where` was not used. same for the next one.

Reply:
I looked at the source code. It's better (safer) to have a `where` clause first but the class initializes it anyway so it will work.